modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2142,10 +2133,60 @@ private void _deleteFileEntries(
 				continue;
 			}
 
-			try {
-				_dlFileEntryLocalService.deleteFileEntry(
+			List<Long> orphanedFileEntryIds = new ArrayList<>();
```

Comment:
We could use the term `stale` > `staleFileEntryIds` ? what do you think?
I could not find orphaned in the portal, and I recently reviewed a PR from Paulo and Igor, and Brian used the term stale for outdated data.

Reply:
I saw that `orphaned` is used for files [here](https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/com/liferay/portlet/documentlibrary/service/persistence/impl/DLFileEntryFinderImpl.java#L399). What do you think?